src/shared/components/Loading/Loading.tsx

```diff
@@ -0,0 +1,67 @@
+import Lottie from 'lottie-web';
+import { useRouter } from 'next/router';
+import nProgress from 'nprogress';
+import { useEffect, useRef, useState } from 'react';
+import { useEffectOnce } from 'react-use';
+
+import * as S from './Loading.styles';
+import { LoadingProps } from './Loading.types';
+
+import loadingLottie from '@src/assets/lotti/loading.json';
+
+const Loading = ({ initialLoadingState }: LoadingProps) => {
```

Comment:
```suggestion
const Loading = ({ initialLoadingState = false }: LoadingProps) => {
```
로 초기값을 할당하고 useState의 ?? 연산자는 제거해도될것같아용!